features/mailbox.feature

```diff
@@ -3,12 +3,21 @@ Feature: Mailbox
   In order to test full user journeys
   I want to be able to emulate Gov Notify's email behaviour
 
-  Scenario: Receiving an email
+  Scenario: Check all emails
     Given an email is sent to Gov Notify to "matt@example.com"
-    When I check the email inbox
-    Then I should see 1 email sent to "matt@example.com"
+    And an email is sent to Gov Notify to "tom@example.com"
+    When I check the global email inbox
+    Then I should see 2 email sent to "matt@example.com"
```

Comment:
this should be tom

Reply:
In fact I lie, it would be 1 email to each. Basically, this line doesn't make sense :P